docs/source/guide/training-edge.rst

```diff
@@ -51,7 +51,8 @@ node representations on each edge.
     import dgl.function as fn
     class DotProductPredictor(nn.Module):
         def forward(self, graph, h):
-            # h contains the node representations computed from the GNN above.
+            # h contains the node representations computed from the GNN defined
+            # in the node classification section.
```

Comment:
"the node classification section" -> "section 5.1 Node Classification/Regression"